chips/litex_vexriscv/src/plic.rs

```diff
@@ -0,0 +1,93 @@
+// TODO: This module may require renaming, as in VexRiscv the
+// interrupt controller probably isn't really called "PLIC"
```

Comment:
Yeah it should be renamed. PLIC is actually a RISC-V spec so you either support it or you don't (lowRISC has a non-compliant PLIC unfortunately).